plat/renesas/rcar/bl2_plat_mem_params_desc.c

```diff
@@ -19,6 +20,8 @@
 #define BL33_MODE MODE_EL2
 #endif
 
+extern uint64_t fdt_blob[PAGE_SIZE_4KB / sizeof(uint64_t)];
```

Comment:
Is the kernel DTB being created in secure memory ? How would BL33 access the secure memory ?

Also, would prefer to use arg1 for the kernel DTB to be inline with Dynamic config framework in TF-A. https://github.com/ARM-software/arm-trusted-firmware/blob/master/docs/firmware-design.rst#11dynamic-configuration-during-cold-boot